sklearn/utils/tests/test_validation.py

```diff
@@ -563,6 +564,14 @@ def test_has_fit_parameter():
     assert_true(has_fit_parameter(SVR, "sample_weight"))
     assert_true(has_fit_parameter(SVR(), "sample_weight"))
 
+    class TestClassWithDeprecatedFitMethod:
+        @deprecated("Deprecated for the purpose of testing has_fit_parameter")
+        def fit(self, X, y, sample_weight=None):
+            pass
+
+    assert_true(has_fit_parameter(TestClassWithDeprecatedFitMethod,
```

Comment:
Please just use bare assert. We're phasing out the use of assert_true, assert_equals, etc.

Reply:
Okay, replaced assert_true with assert.